### PR TITLE
Answers for week 7

### PR DESCRIPTION
**Pull request by morarodgers**
Completed the assignment

### CODE REVIEW

File: 04-store-api/starter/controllers/products.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    }
+    else {
+        result = result.sort('createdAt')
+    } 
+    if (fields) {
+        const fieldList = fields.split(',').join(' ');
+        result = result.sort(fieldList)
+    }
+    const page =Number(req.query.page) || 1
+    const limit = Number(req.query) || 10
```
Comment (joseluiz) [inline 04-store-api/starter/controllers/products.js]: You have to add limit here. Ex: req.query.limit

File: 04-store-api/starter/controllers/products.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    if (sort) {
+        const sortList = sort.split(',').join(' ');
+        result = result.sort(sortList)
+    }
+    else {
+        result = result.sort('createdAt')
+    } 
+    if (fields) {
+        const fieldList = fields.split(',').join(' ');
+        result = result.sort(fieldList)
```
Comment (joseluiz) [inline 04-store-api/starter/controllers/products.js]: You need to use the select method here. Ex: result.select(fieldList)

**Review by joseluiz — CHANGES_REQUESTED** [review]
Hi Rodger. I found a few things you need to fix. Please see the PR comments for details. If you have any questions please call me on Slack.

File: 04-store-api/starter/controllers/products.js (hunk 1 above)
Comment (morarodgers) [reply 04-store-api/starter/controllers/products.js]: Okey working on it

File: 04-store-api/starter/controllers/products.js (hunk 2 above)
Comment (morarodgers) [reply 04-store-api/starter/controllers/products.js]: Alright, on it

**Review by joseluiz — APPROVED** [review]
PR approved. Good Job!

**Pull request merged**